tests/test_feedexport.py

```diff
@@ -7,15 +7,15 @@
 import tempfile
```

Comment:
Please revert the changes to the import section of this file, these changes broke PEP8-compliance.